src/compiler/app-core/app-polyfills.ts

```diff
@@ -2,7 +2,11 @@ import { join } from 'path';
 
 import type * as d from '../../declarations';
 
-export const getClientPolyfill = async (config: d.Config, compilerCtx: d.CompilerCtx, polyfillFile: string) => {
+export const getClientPolyfill = async (
```

Comment:
_Just_ brought us over the line limit by a hair by going from `Config`->`ValidatedConfig`